runtime/call_metrics.go

```diff
@@ -165,7 +165,9 @@ func NewInboundHTTPMetrics(scope tally.Scope) *InboundHTTPMetrics {
 	metrics.Errors = scope.Counter(inboundCallsErrors)
 	metrics.Status = make(map[int]tally.Counter, len(knownStatusCodes))
 	for statusCode := range knownStatusCodes {
-		metrics.Status[statusCode] = scope.Counter(fmt.Sprintf("%s.%d", inboundCallsStatus, statusCode))
+		metrics.Status[statusCode] = scope.Tagged(map[string]string{
+			"status": fmt.Sprintf("%d", statusCode),
+		}).Counter(fmt.Sprintf("%s.%d", inboundCallsStatus, statusCode))
```

Comment:
If we are adding status as a tag, do we need the `statusCode` suffix in the counter name?

Reply:
The tag is only there because M3QL does not implement `aliasSub()`.

We can keep the statusCode in the metric name for more flexible queries